polaris-react/src/components/ResourceItem/ResourceItem.stories.tsx

```diff
@@ -78,6 +78,22 @@ export function SelectableWithMedia() {
             name: 'Yi So-Yeon',
             location: 'Gwangju, South Korea',
           },
+          {
```

Comment:
Added more items to test focus ring border styles for first/second/last children for `ResourceItem`.